modules/rest-api/src/main/java/org/apache/ignite/internal/rest/api/recovery/DisasterRecoveryApi.java

```diff
@@ -41,85 +41,6 @@
 @Controller("/management/v1/recovery")
 @Tag(name = "recovery")
 public interface DisasterRecoveryApi {
-    @Get("state/local")
-    @Operation(operationId = "getLocalPartitionStates", description = "Returns local partition states.")
```

Comment:
Probably, I've missed that. We still have an ability to retrieve local/global partition states, where partition states are zone partition states, right?

Reply:
Yes, local and global states are still available:
```
    @Get("zone/state/local")
    CompletableFuture<LocalZonePartitionStatesResponse> getZoneLocalPartitionStates()...

    @Get("zone/state/global")
    CompletableFuture<GlobalZonePartitionStatesResponse> getZoneGlobalPartitionStates()...
```